docs/docs-graphql/schema.graphqls

```diff
@@ -2837,6 +2847,13 @@ type Query {
   version: String!
 }
 
+"""Expected return type of an execution"""
+enum ReturnType {
+  SUCCESS
+  FAILURE
+  ANY
```

Comment:
Isn't it useful to document these values? We don't seem to do it for the core types, but we do in `TypeDefKind` for example. Would show in user's IDE.

Reply:
Oop, yup!